### DISCUSSION TITLE
Autonomous SC triggers on the amount of fees

### DISCUSSION THREAD

**Discussion by loonggamma** [massalabs/massa#4665] [Ideas]
**Is your feature request related to a problem? Please describe.**
I don't see a fully clear use-case yet, but here's the idea: ASC triggers bring a solution to the on-chain automation, especially in the financial speculation industry. I can assume that they'd be used by many actors in multiple contexts. We have 32 threads with and slot-spots each 0.5sec. 
I don't know how likely it is, but it's possible that actors would be competing for a slot place to execute their async calls. In this case, they will require a trigger conditioned to the amount of transaction fees a an async call should be executed at/below/above/or in range. 
Another point is that, if the call is not included in a slot it was conditioned for, because of the lower tx fees than other actors competing for the slot space placed, how can a user re-condition the execution of the async call? 

**Describe the solution you'd like**
n/a
**Describe alternatives you've considered**
n/a
**Additional context**

**Comment by AurelienFT** [comment]
When using `sendMessage()` you have a parameter fee so your ASC will be executed only if the minimum fee in a slot is less or equal to this number.

For now it's not possible to reconditioned or pretty complicated (maybe need a doc on that and clarify on our side also). You can put big ranges and it will likely be included (or could gone out of the pool before)

@damip do you have ideas on autonomous sc re-conditioning even at SC level ?
